app/views/layouts/_logged_in_header.html.haml

```diff
@@ -0,0 +1,5 @@
+%header.sticky.top-0.left-0.right-0.flex.justify-between.items-center.py-2.px-3.max-w-dvw.overflow-scroll.h-8.z-100.border-b-1.border-neutral-800.text-base{class: 'text-(--main-text-color) bg-(--main-bg-color,var(--background-color))'}
+  .flex-1.truncate{title: current_user.email}= current_user.email
+  %div.flex.gap-4
+    %div= link_to('My account', my_account_path, class: 'text-inherit!')
+    %div= button_to('Sign out', destroy_user_session_path, method: :delete)
```

Comment:
Styling this logged-in header entirely with Tailwind allows us to render and style it on the blog without needing to pull in an additional stylesheet (beyond the Tailwind styles and other global styles that we were already pulling in for show pages since https://github.com/davidrunger/david_runger/pull/6599 and for the index page as of this PR).